tsconfig.json

```diff
@@ -36,6 +36,7 @@
 			"@umbraco-cms/backoffice/repository": ["libs/repository"],
 			"@umbraco-cms/backoffice/resources": ["libs/resources"],
 			"@umbraco-cms/backoffice/router": ["libs/router"],
+			"@umbraco-cms/backoffice/sorter": ["libs/sorter"],
```

Comment:
This should be added to web-test-runner.config.mjs as well and make sure to tell me when it's merged or put it on the v13/dev branch in the importmap as well.